lib/bundler_api/cdn.rb

```diff
@@ -7,6 +7,7 @@ def self.purge_specs(client = self.client)
     client.purge_path '/latest_specs.4.8.gz'
     client.purge_path '/specs.4.8.gz'
     client.purge_path '/prerelease_specs.4.8.gz'
+    client.purge_path "/names"
```

Comment:
I put those on the versions commit. But it is on a [different place](https://github.com/bundler/bundler-api/pull/78/files#diff-ef593641247bb94fa6b5b8e170888259R27) - It is in other place because this should be purged if someone runs the rake task to create a new file or update the current one